src/wstore/admin/views.py

```diff
@@ -20,18 +20,20 @@
 
 from __future__ import unicode_literals
 
-import json
+from wstore.store_commons.resource import Resource
+from wstore.store_commons.utils.http import authentication_required, JsonResponse
+from wstore.store_commons.utils.units import ChargePeriod, CurrencyCode
 
-from django.http import HttpResponse
 
-from wstore.store_commons.utils.http import authentication_required
-from wstore.store_commons.utils.units import supported_currencies
-from wstore.store_commons.resource import Resource
+class ChargePeriodCollection(Resource):
+
+    @authentication_required
```

Comment:
Are you sure that this is a restricted API? If this info is retrieved when the users are accessing the home portal, which allows to get catalog info being anonymous, you are going to see an error in the portal